src/main/java/com/overops/plugins/sonar/measures/OverOpsQualityGateStat.java

```diff
@@ -0,0 +1,100 @@
+package com.overops.plugins.sonar.measures;
+
+import com.takipi.api.client.functions.output.EventRow;
+import com.takipi.api.client.functions.output.RegressionRow;
+import com.takipi.api.client.functions.output.ReliabilityReport;
+import com.takipi.api.client.functions.output.ReliabilityReportRow;
+
+import java.util.*;
+
+public class OverOpsQualityGateStat {
```

Comment:
I believe your logic for new, critical, and increasing is correct based on this example: https://github.com/takipi/api-client/blob/ad87f5d912077d3800935808b1a40c5b96dc5707/api-client-util/src/main/java/com/takipi/api/client/functions/sample/OverOpsFunctionsSample.java

I think your logic for resurfaced is also correct, but have not yet been able to verify this.

Reply:
@daverted Yes, I use the same approach to retrieve

>  new, critical, and increasing